app/(afterLogin)/(deal)/deals/manage-deals/page.tsx

```
@@ -0,0 +1,122 @@
+'use client';
+import React, { useState } from 'react';
+import { Input, Button, Typography, Dropdown } from 'antd';
+import { SearchOutlined, PlusOutlined } from '@ant-design/icons';
+import { RiExchange2Line } from 'react-icons/ri';
+
+import KanbanBoard from './_components/kanbanBoard';
+import DealSideBar from './_components/sideBar';
+import FilterModal from './_components/filter';
+import { LuSettings2 } from 'react-icons/lu';
+import FilterDropdown from '@/app/(afterLogin)/reports/_components/filterDropdown';
```

ESLint

[error] 'FilterDropdown' is defined but never used.

(@typescript-eslint/no-unused-vars)